infra/lib/logging/stack.ts

```diff
@@ -0,0 +1,162 @@
+import { Construct } from "constructs";
+import { IntegrationTestStackEnvironment, IntegrationTestStackEnvironmentProps, AmplifyCategory, IntegrationTestStack, randomBucketName } from "../common";
+import * as cdk from "aws-cdk-lib"
+import * as logs from "aws-cdk-lib/aws-logs"
+import * as path from "path"
+import * as iam from "aws-cdk-lib/aws-iam"
+import * as cognito_identity from "@aws-cdk/aws-cognito-identitypool-alpha";
+import { BucketDeployment, Source } from "aws-cdk-lib/aws-s3-deployment";
+
+import * as S3 from "aws-cdk-lib/aws-s3"
+import * as lambda from "aws-cdk-lib/aws-lambda"
+import * as ApiGateway from "aws-cdk-lib/aws-apigateway"
```

Comment:
Use same casing for import aliases. CDK docs use snake case e.g. `cognito_identity`, `iam`, `api_gateway`, etc. so that's what we use.